docs/20-mongodb-atlas/2-create-account.mdx

```diff
@@ -3,7 +3,29 @@ import Screenshot from "@site/src/components/Screenshot";
 
 # 👐 Create Your Account
 
-To use MongoDB Atlas, start by creating an account. If you already have an account, you can skip this step.
+To use MongoDB Atlas, start by creating an account.
+
+<details>
+  <summary>🦸‍♀️ If you already have an Atlas account, you can use it by creating a new project</summary>
+  
+    First, log into <Link to="https://account.mongodb.com/">MongoDB Atlas</Link> using your existing credentials.
+
+    <Screenshot url="https://account.mongodb.com" src={"img/screenshots/20-mongodb-atlas/2-create-account/3-login.png"} alt="Login form" />
+
+    ## Create a new Atlas project
```

Comment:
@nestor-daza-mdb do you need to create a new project if you're also creating your account for the 1st time? This is good info, but only useful for those who already have a MongoDB account and a free M0 cluster in their project. Most don't know they need to create a separate project

Reply:
I think this should have a `:::info` [admonition](https://docusaurus.io/docs/next/markdown-features/admonitions) to say when is this applicable: when you already have a free M0 cluster

Reply:
I included this block as an expandable section (_details_, instead of _admonition_), closed by default.  The summary (the only visible part when the page loads) says this only applies "If you already have an Atlas account". A new project will have an M0 available, regardless of other existing projects in the account